client/app/lib/onboarding.coffee

```diff
@@ -98,8 +98,7 @@ class Step
 
     # Error handler for save() call
     handleSaveError: (err) =>
-        error = Object.values err.errors
-        throw new Error error
+        throw new Error err.error
```

Comment:
According to the [error format use on the server side](https://github.com/cozy/cozy-proxy/blob/development/server/middlewares/errors.coffee#L20) (that I didn't change with my PR #322 ), ```errors``` property could be missing from the server result. It's why I prefer to use here ```err.error``` instead of ```err.errors```. This ```err.error``` message can also be translated after.